drivers/os/linux/linux.go

```diff
@@ -67,6 +67,13 @@ func (d *driver) Mounts(
 		return nil, goof.New("cannot specify mountPoint and deviceName")
 	}
 
+	// TODO: s3fs doesnt provide mounted bucket in /proc/self/mountinfo
+	// that's why m.Source is just 's3fs'
+	// so, it is workaround to patch Source info
+	// Another option - use moinPoint to filter mounts into
+	// the integration driver logic and remove this patching
+	d.patchMounts(ctx, &mounts)
```

Comment:
@akutz I agree that the patching should happen in the executors `LocalDevices()`. I am not following, however, what would need to be added to the integration driver that is related to this... I feel like I should see it, but I don't.

With your work to add mount/unmount functionality capability to the executors, what would need to be added to the integration driver? I'm assuming a check to see if the volume we are working with needs to be mounted via executor, or one of our existing "standard" mount functions, but is there anything else related to the patching of mount info?